packages/serverless/src/apollo/app/rollup.ts

```diff
@@ -0,0 +1,61 @@
+import Joi from 'joi';
+import { authorizeWrapper } from '../validation.js';
+import { databaseName, transactionType } from './common.js';
+import { TDatabaseRequest } from './database.js';
+import GraphQLJSON from 'graphql-type-json';
+import { getRollUpHistory as getRollUpHistoryDomain} from '../../domain/use-case/rollup.js';
+
+export const typeDefsRollUp = `#graphql
+scalar Date
+type Mutation
+
+enum RollUpStatus {
+  update
+  outdated
+  failed
+}
+
+type RollUpHistoryItem {
+  databaseName: String!
+  transactionType: TransactionType!
+  txHash: String!,
+  status: TransactionStatus!,
+  currentMerkleTreeRoot: String!,
+  previousMerkleTreeRoot: String!,
+  createdAt: Date!
+  error: String
+}
+
+type RollUpHistory {
+  status: RollUpStatus!,
+  statusData: Int,
```

Comment:
is that extraData. This type dont compatible with getRollUpHistoryDomain

Reply:
done